runtime/lua/vim/treesitter/highlighter.lua

```diff
@@ -97,7 +97,10 @@ function TSHighlighter.new(tree, opts)
   if vim.g.syntax_on ~= 1 then
     vim.api.nvim_command('runtime! syntax/synload.vim')
```

Comment:
Might want to change that to `a.nvim_command` here as well while you're at it :)

Reply:
That could even be `vim.cmd`